vcd/datasource_vcd_nsxt_edgegateway_qos_profile_test.go

```diff
@@ -0,0 +1,66 @@
+//go:build ALL || nsxt || gateway || functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccVcdDatasourceNsxtGatewayQosProfile(t *testing.T) {
+	skipIfNotSysAdmin(t)
+
+	vcdClient := createTemporaryVCDConnection(true)
```

Comment:
Why not `createTemporaryVCDConnection(false)`? If I understood, we always require the client to check VCD version, don't we?
For example, if it returns nil, then
```
if vcdClient == nil {
		t.Skip(acceptanceTestsSkipped)
	}
```
would skip the test, even if we are in version >= 36.2, which we don't want, do we?

Reply:
Double checking this. 
I added such a combination to avoid early exit and still generate binary test file (for versions 10.3.2+), but rechecking it.

```
	if vcdShortTest {
		t.Skip(acceptanceTestsSkipped)
		return
	}
```